test/e2e/test/maps/builder.go

```diff
@@ -148,6 +148,17 @@ func (b Builder) MutationReversalTestContext() test.ReversalTestContext {
 }
 
 func (b Builder) SkipTest() bool {
+	// only execute EMS tests if we have a test license to work with
+	if test.Ctx().TestLicense == "" {
+		return true
+	}
+
+	// TODO: remove once an EMS 8.x Docker image image is available
```

Comment:
nit: Should we also create an issue ?

Reply:
https://github.com/elastic/cloud-on-k8s/issues/4479